IdentityCore/tests/integration/MSIDCacheSchemaValidationTests.m

```diff
@@ -791,7 +801,9 @@ - (void)testSchemaComplianceForRefreshToken_whenMSSTSResponse_withB2CAccountAndT
                                    @"secret": @"<removed_rt>",
                                    @"environment": @"login.microsoftonline.com",
                                    @"credential_type": @"RefreshToken",
-                                   @"home_account_id": @"ad020f8e-b1ba-44b2-bd69-c22be86737f5-b2c_1_signin.ba6c0d94-a8da-45b2-83ae-33871f9c2dd8"
+                                   @"home_account_id": @"ad020f8e-b1ba-44b2-bd69-c22be86737f5-b2c_1_signin.ba6c0d94-a8da-45b2-83ae-33871f9c2dd8",
+                                   @"not_before": @1538801260,
```

Comment:
@oldalton are you okay with this? These are numbers, not strings

Reply:
Is not before and refresh_token_expires_in part of the schema?

Reply:
I do not see them in the doc, so I don't think so. I am not sure what they're for - looks like they're specific to the b2c scenario? Should they be serialized?

If the answer is "yes", I think we need to add them to the schema doc.

If the answer is "no", I think they shouldn't be in the additional fields on the credential object.

Sooner or later, the apps will want to store their data in the universal storage, and we will need to put this data in the additional fields, and serialize these fields when writing to disk. Additional fields are also used to write all the fields from disk we don't understand. In other words, I think the additional fields represent the information which doesn't fit into primary fields, but does get written to and read from the disk. We shouldn't be using them to store information we don't want to serialize, and we definitely don't want to mix serializable and non-serializable fields in them (in my opinion).

What do you think?

Reply:
Also, AccountCacheItem does serialize all additional fields:
```
    if (_additionalAccountFields)
    {
        [dictionary addEntriesFromDictionary:_additionalAccountFields];
    }
```
I think it would make a lot of sense for them to be consistent.